iothub/device/src/Common/Extensions/ExceptionExtensions.cs

```diff
@@ -50,6 +50,10 @@ public static IEnumerable<TException> Unwind<TException>(this Exception exceptio
             return exception.Unwind().OfType<TException>();
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage(
+            "Performance",
+            "CA1825:Avoid zero-length array allocations",
+            Justification = "net451 does not support Array.Empty<T>")]
```

Comment:
Or you can do as we have elsewhere, where you have an `#if NET451` directive and change the type based on framework. Then later when we deprecate net451, we just remove the relevant directives.

Reply:
done.